doc/install.text

```diff
@@ -0,0 +1,197 @@
+
+conda install gcc
+
+
+* For convenience of this quick-start guide, create a directory for the PFS-DRP repositories:
+
+export PFS_DRP=<your_target_directory>
+(e.g. export PFS_DRP=$HOME/PFS
+mkdir -p $PFS_DRP
+cd $PFS_DRP
+
+
+* Clone the relevant git repositories. Note that currently a newer version of sconsUtils
+  is needed to suppress some irrelevant compiler warnings. Installing and setting up
+  sconsUtils will be obsolete in the next binary LSST Stack installation.
+
+git clone https://github.com/Subaru-PFS/drp_stella.git
+git clone https://github.com/Subaru-PFS/drp_stella_data.git
+git clone https://github.com/Subaru-PFS/obs_pfs.git
+git clone git@github.com:lsst/sconsUtils.git
+
+* The next 2 steps are temporary workarounds until fixed:
+* Set $EUPS_PKGROOT
+
+export EUPS_PKGROOT="https://sw.lsstcorp.org/eupspkg/"
+
+
+* Install Eigen with the unsupported modules:
+
+eups distrib install eigen 3.2.5.lsst2
+
+
+* Build and setup the pipeline. Note that setting up drp_stella will overwrite
+  the already set up sconsUtils version with the standard one, so we need to
+  setup sconsUtils again before compiling drp_stella. There are a few warnings
+  when compiling drp_stella, one type of warnings is a doxygen warning "... not
+  found in argument list..." and the other is a SWIG problem ("...destination for
+  this 'memset' call is a pointer..."). Both types of warnings will be silenced 
+  in future version and can be safely ignored.
+
+cd $PFS_DRP/sconsUtils
+git checkout w.2016.34
+setup -r .
+scons -Q opt=3 -j8 --filterWarn
+
+cd $PFS_DRP/obs_pfs
+setup -r .
+scons -Q opt=3 -j8 --filterWarn
+
+cd $PFS_DRP/drp_stella_data
+setup -r .
+
+cd $PFS_DRP/drp_stella
+setup -r .
+setup -r ../sconsUtils
+scons -Q opt=3 -j8 --filterWarn
+
+
+* Now for using the pipeline. 
+  Raw test data are in $DRP_STELLA_DATA_DIR/tests/data/raw/:
+    - 3 Biases: visit numbers 7251-7253,
+    - 3 Darks: visit numbers 7291-7293,
+    - 1 Flat: visit number 5,
+    - 1 Arc: visit number 4 
+  Configuration parameters for the pipeline tasks can be set either in config files
+  (see $OBS_PFS_DIR/config/pfs/) or by passing them on the command line (after 
+  "--config", e.g. --config isr.doDark=False). You can list all configuration
+  parameters by appending a "--show config" to the parameter list. Setting "--show
+  config=glob" 
+
+
+
+
+
+
+
+
+
+
+
+
+  First we need to create a directory (actually 2) where we want to store pipeline outputs. 
+  Let's assume you want to store the pipeline outputs in a directory $HOME/spectra/PFS.
+  For the convenience of this quick-start guide we define another environment variable:
+
+export PFS_DATA=$HOME/spectra/PFS
+mkdir -p $PFS_DATA/CALIB
+
+
+* We need to tell the LSST stack which mapper to use. The mapper gives a logical view
+  of both the raw data and the pipeline outputs which you can query for particular data
+  sets. It abstracts out the physical layout so we don't have to worry about the details.
```

Comment:
Can one query a view? Hmm.

How about: "The mapper provides a logical view of both the raw data and pipeline outputs, and provides facilities for querying for particular data sets. It abstracts away the details of the underlying storage, so we can avoid worrying about implementation details."?